src/app/chip_data_model.gni

```diff
@@ -131,6 +131,7 @@ template("chip_data_model") {
     deps += [
       ":${_data_model_name}_zapgen",
       "${chip_root}/src/app/common:cluster-objects",
+      "${chip_root}/src/app/common:enums",
```

Comment:
This really shouldn't be needed, right?  :cluster-objects should have :enums in public_deps, since cluster-objects.h includes the enums header....

Reply:
seems indeed not needed (at least on some test compiles). Removed it.